src/sentry/incidents/logic.py

```diff
@@ -973,6 +1010,22 @@ def update_alert_rule_trigger(
             ]
             AlertRuleTriggerExclusion.objects.bulk_create(new_exclusions)
 
+        if actions is not None:
+            # Delete actions we don't have present in the updated data.
+            action_ids = [x["id"] for x in actions]
```

Comment:
Same thing here, I think this can error for rows with no `id`

Reply:
Good catch!